packages/components/src/components/input/input.jsx

```diff
@@ -24,7 +24,9 @@ const Input = (
     const [counter, setCounter] = React.useState(0);
 
     React.useEffect(() => {
-        if (initial_character_count) setCounter(initial_character_count);
+        if (typeof initial_character_count === 'number') {
```

Comment:
How about adding `PropTypes`?

Reply:
Sure, that should happen, however it won't solve the problem here. The issue is that `0` is falsy so the check doesn't work.

Reply:
True, I meant **also** adding `PropTypes`.